client/src/components/CityInformation.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import {
+  useParams,
+} from 'react-router-dom';
+import httpService from '../services/httpService';
+import Img from './common/Img';
+import ParagraphText from './common/Paragraph';
+import HeaderTitle from './common/Title';
+import './city.css';
+
+function CityInformation() {
+  const { id } = useParams();
+  const [cityInfo, setcityInfo] = useState({});
+  const [families, setfamilies] = useState([]);
+  useEffect(() => {
```

Comment:
we'll fetch data from the city page, check cityProvider and get data from there

Reply:
done